### ISSUE TITLE
The keyboard focus remains on the tree view elements of the Left pane (Tree View pane) of Inspector window for the double hit of the Tab keys.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#291]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to Left pane (Tree View pane) of the inspector window.
3. Traverse the entire window using tab key.
4. Observe the keyboard behavior while focus is on the any of  the tree view elements of the left pane.
### Expected Behavior
Once user hit on tab key to traverse the interacative elements of the window, the keyboard focus order should be proper and logical.
### Actual Behavior
The keyboard focus remains on the tree view elements of the Left pane (Tree View pane)  for the double hit of the Tab keys.
Note: Although the focus is landing from UI tree elements to the Main pane(Frame and Contents pane) for the second hit of the tab key, but focus is not visible on the Main pane.
### User Impact
This might cause confusion and difficulty to find the actual keyboard focus for the elements.

![kb7_leftpaneui-tree_focusremainsonuitreefordoublehitoftab](https://user-images.githubusercontent.com/22956130/39987112-656442ae-5781-11e8-8c4a-bc3705d6682d.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.4.3

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
